code/modules/mob/living/carbon/human/human.dm

```diff
@@ -300,6 +300,7 @@
 			if(pocket_item)
 				if(pocket_item == (pocket_id == SLOT_R_STORE ? r_store : l_store)) //item still in the pocket we search
 					dropItemToGround(pocket_item)
+					usr.put_in_hands(pocket_item)
```

Comment:
should probably remove `dropItemToGround(pocket_item)` to make it truely stealthy, otherwise the drop animation will still play

Reply:
> should probably remove `dropItemToGround(pocket_item)` to make it truely stealthy, otherwise the drop animation will still play

I didn't see the item show when I tested this, and when I originally tried this without having dropItemToGround it created a copy of the item that acted like the target's pocket and my hand were now paradox bags holding that item.

Reply:
weird, fair enough, byond code is more jank than i thought i guess.